doc/going-further/symfony-task.md

```diff
@@ -0,0 +1,34 @@
+# Symfony Task
+
+If castor lives near a Symfony application, you may want to use directly some
+commands. This is possible with the `AsSymfonyTask` attribute:
+
+```php
+#[AsCommand('hello', 'Says hello from a symfony application')]
+#[AsSymfonyTask(name: 'symfony:hello', console: [__FILE__])]
```

Comment:
I think this example could more helpful. IMO it should provide a realistic example (like with `bin/console`))

Reply:
It means we have to embed a whole Symfony project just for one or two commands?
It does not worth it, isn't?

Reply:
Not sure to understand, I believe documentation code sample does not need to be the same as the `examples` directory but must reflect real world usages

Reply:
ahhh, got it, this is the documentation! I'll fix it